nautobot/core/ui/object_detail.py

```diff
@@ -901,18 +901,28 @@ def render_body_content(self, context):
 class _BaseTextPanel(Panel):
     """A panel that renders simple text or markdown"""
 
+    RENDER_OPTIONS = namedtuple("RENDER_OPTIONS", ["text", "json", "yaml", "markdown"])
+
     def __init__(
-        self, *, enable_markdown=True, body_content_template_path="components/panel/body_content_text.html", **kwargs
+        self,
+        *,
+        render_as=RENDER_OPTIONS.markdown,
```

Comment:
I feel like plaintext might be a more intuitive default?

Reply:
In couple of places I've seen that our textarea has built-in support for markdown and I assumed that this is most common than just plaintext.